Vungle/VungleRouter.m

```diff
@@ -641,6 +711,7 @@ - (void) vungleSDKDidInitialize
 
 - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
```

Comment:
Note: 
This is the new callback for header bidding. It is not implemented yet in the native SDK.